src/layouts/contexts/NetworkContext.tsx

```diff
@@ -2,19 +2,42 @@ import { getEnvironment } from '@contexts/Environment'
 import { useRouter } from 'next/router'
 import { createContext, PropsWithChildren, useContext, useState } from 'react'
 
+export declare type NetworkName = 'mainnet' | 'testnet' | 'regtest'
+
 export enum Network {
   LocalPlayground = 'Local',
   RemotePlayground = 'Playground',
   MainNet = 'MainNet',
   TestNet = 'TestNet'
 }
 
+export interface NetworkObject {
+  name: NetworkName
+}
+
 const NetworkContext = createContext<Network>(undefined as any)
 
 export function useNetwork (): Network {
   return useContext(NetworkContext)
 }
 
+export function useNetworkObject (): NetworkObject {
```

Comment:
A few ideas:

0. you can separate out this change into another PR
1. `import { Network as NetworkObject } from '@defichain/jellyfish-network'`
2. or alternatively you can rename `export enum Network` to export enum `ConnectedNetwork`
3. deprecate `useNetwork()`
4. `export function useNetworkObject (): NetworkObject `

Reply:
Yea it's getting quite confusing now. I will write an issue later to bounce off some ideas with the team.